components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/notifier/MarketplaceAssistantApiPublisherNotifier.java

```diff
@@ -167,11 +164,11 @@ private void deleteRequest(APIEvent apiEvent) throws NotifierException {
 
         try {
             String uuid = apiEvent.getUuid();
-            APIUtil.DeleteApi(marketplaceAssistantConfigurationDto.getEndpoint(),
+            APIUtil.deleteApi(marketplaceAssistantConfigurationDto.getEndpoint(),
                     marketplaceAssistantConfigurationDto.getAccessToken(), marketplaceAssistantConfigurationDto.getApiDeleteResource(), uuid);
         } catch (APIManagementException e) {
-            String errorMessage = "Error encountered while Deleting the API from the vector database";
-            log.error(errorMessage);
+            String errorMessage = "Error encountered while Deleting the API from the vector database" + e.getMessage();
```

Comment:
Let's add the API name. or some identifier. Please check other places as well

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/12385/commits/e23b99dea51b29c978fed6b6fefb746969f086e8